rust/included_store/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2025 Arista Networks, Inc.
+// Use of this source code is governed by the Apache License 2.0
+// that can be found in the LICENSE file.
+#![deny(unused_crate_dependencies)]
+
+/// The full AVD schema is pre-compiled into this crate as a bytestream of XZ2 compressed JSON.
+/// Include it from various bindings and cache it with OnceLock like:
+/// ```
+/// use included_store::get_store as get_included_store;
+///
+/// static STORE: OnceLock<Store> = OnceLock::new();
```

Comment:
```suggestion
/// The full AVD schema is pre-compiled into this crate as a bytestream of GZ compressed JSON.
/// Include it from various bindings and cache it with OnceLock like:
/// ```
/// use std::sync::OnceLock;
/// use included_store::get_store as get_included_store;
/// use avdschema::Store;
///
/// static STORE: OnceLock<Store> = OnceLock::new();
```